lib/src/configuration.dart

```diff
@@ -54,6 +54,11 @@ typedef InitialDataCallback = void Function(Realm realm);
 /// The `oldSchemaVersion` argument indicates the version from which the Realm migrates while
 typedef MigrationCallback = void Function(Migration migration, int oldSchemaVersion);
 
+/// The signature of a callback that will be triggered when a Client Reset error happens in a synchronized Realm.
+///
+/// The `clientResetError` argument holds useful data to be used when trying to manually recover from a client reset.
+typedef ClientResetCallback = void Function(SyncClientResetError clientResetError);
```

Comment:
Why do we call the type `SyncClientResetError` now. Should we rename to ClientResetError handler? @nirinchev ?

Reply:
I will rename it